docker/Dockerfile.deepstream

```diff
@@ -45,7 +45,7 @@ RUN python setup.py bdist_wheel && rm -rf _skbuild
 FROM base_builder AS savantboost_build
 
 # newer cmake ver. needed for "FindCUDAToolkit"
```

Comment:
Do we still need `newer cmake ver. for "FindCUDAToolkit"`?

Reply:
Deleted this comment